lua/competitest/receive.lua

```diff
@@ -149,24 +151,44 @@ end
 ---Utility function to store received problem (source file and testcases)
 ---@param filepath string: source file absolute path
 ---@param template_file string | nil: template file absolute path, or nil to create empty source file
+---@param evaluate_template boolean: whether to evaluate the template for receive modifiers
+---@param task table: table with all task details
 ---@param tcdir string: directory where testcases files will be stored
----@param tclist table: table containing received testcases
 ---@param use_single_file boolean: whether to store testcases in a single file or not
 ---@param single_file_format string: string with CompetiTest file-format modifiers to match single testcases file name
 ---@param input_file_format string: string with CompetiTest file-format modifiers to match input files name
 ---@param output_file_format string: string with CompetiTest file-format modifiers to match output files name
-function M.store_problem(filepath, template_file, tcdir, tclist, use_single_file, single_file_format, input_file_format, output_file_format)
+---@param date_format string: string used to format date
+function M.store_problem(
```

Comment:
I'm going to remove this function since it's only called by `store_problem_config`, initially it was meant to be used outside it too.